nilmtk/dataset_converters/dred/Readme.md

```diff
@@ -6,3 +6,15 @@ DRED (Dutch Residential Energy Dataset) is the first open-access, publicly avail
 * Household information - house layout, number of appliance monitored, appliance-location mapping, etc.
 
 The [data set homepage](http://www.st.ewi.tudelft.nl/~akshay/dred/) contains nilmtk H5 file, in addition to the raw data.
+
+
+UPDATES :-
```

Comment:
Please remove the link to the generated H5. I will check with the author if they'd like to update the H5.

Reply:
Okay. I am removing the generated .h5 link